mm2src/coins/utxo/utxo_common.rs

```diff
@@ -2606,15 +2606,15 @@ where
     let now = now_ms() / 1000;
     if now < locktime {
         let to_wait = locktime - now + 1;
-        return Box::new(futures01::future::ok(CanRefundHtlc::HaveToWait(to_wait)));
+        return Box::new(futures01::future::ok(CanRefundHtlc::HaveToWait(to_wait.max(3600))));
```

Comment:
Is it always 1 hour for all coins? As far as I understand, the locktime can be more than `swap_start_time + 4 hours`. Correct me please if I'm wrong.

Reply:
The `to_wait` determines a time to wait before the next `can_refund_now` call in the loop.

https://github.com/KomodoPlatform/atomicDEX-API/blob/9f6398ba48539a26457695ab26f33d605c8e4468/mm2src/lp_swap/maker_swap.rs#L774-L783

 So even lock time is much larger MM2 won't try to refund the payment after 1 hour.

Reply:
I see, thank you for the answer!